### PR TITLE
feat: account details page

### LINKED ISSUE #498: Account Details Page Front End
https://www.figma.com/file/VV63sXC9UbGAUczGOEOJVapn/Solana-Network-Explorer-MVP?node-id=322%3A2

### PR DESCRIPTION
**Pull request by manuel-calavera**
Fixes #498

### CODE REVIEW

File: src/AppV2.js [hunk 1]
```diff
@@ -132,6 +133,7 @@ const App = () => {
                   component={ApplicationDetail}
                 />
                 <Route exact path="/favorites" component={Favorites} />
+                <Route exact path="/account/:id" component={AccountDetail} />
```
Comment (sunnygleason) [inline src/AppV2.js]: 👍

File: src/v2/components/Account/Detail/Chart/index.jsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  return (
+    <div className={classes.tooltip}>
+      <div className={classes.tooltipDot} />
+      <div className={classes.tooltipTitle}>AVG TPS:{data.y}</div>
+      <div className={classes.tooltipDate}>{data.date}</div>
+    </div>
+  );
+};
+
+const Chart = props => {
```
Comment (sunnygleason) [inline src/v2/components/Account/Detail/Chart/index.jsx]: 👍 (we don't have the data for this chart yet)

File: src/v2/components/Account/Detail/Code/index.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+// @flow
+import React from 'react';
+import HelpLink from 'v2/components/HelpLink';
+import CopyBtn from 'v2/components/UI/CopyBtn';
+import {observer} from 'mobx-react-lite';
+import YAML from 'yaml';
+
+import useStyles from './styles';
+
+const AccountCode = ({accountView}: {accountView: Object}) => {
```
Comment (sunnygleason) [inline src/v2/components/Account/Detail/Code/index.jsx]: 👍

File: src/v2/components/Account/Detail/Transactions/index.jsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    block: '7887219',
+    time: '55 sec ago',
+    timeType: 'in',
+    application_id: '5CpdpKwKUBJgD4Bdase123as12asd21312',
+    type: 'other',
+    confirmations: 5,
+  },
+];
+
+const Transactions = ({transactions}: {transactions: Array}) => {
```
Comment (sunnygleason) [inline src/v2/components/Account/Detail/Transactions/index.jsx]: :+1:

File: src/v2/components/Account/Detail/index.jsx [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+  const specs = [
+    {
+      label: 'Balance',
+      hint: '',
+      value: `${(accountInfo.lamports * LAMPORT_SOL_RATIO).toFixed(8)} SOL`,
+    },
+    {
+      label: 'Transactions',
+      hint: '',
+      value: `${(accountInfo.lamports * LAMPORT_SOL_RATIO).toFixed(8)} SOL`,
```
Comment (sunnygleason) [inline src/v2/components/Account/Detail/index.jsx]: thinking maybe is this supposed to be some other data point? 😕

File: src/v2/components/Transactions/Detail/Application/index.jsx [hunk 6]
```diff
@@ -18,8 +19,10 @@ const Application = ({id, accounts}: TApplication) => {
   const renderAccount = (account, i) => (
     <div className={classes.account}>
       <Label text={`Account ${i + 1}`} hint="" />
-      <Avatar avatarUrl="" name="" width={33} height={33} pubkey={id} />
-      <div className={classes.address}>{account}</div>
+      <Link to={`/account/${id}`} className={classes.accountLink}>
+        <Avatar avatarUrl="" name="" width={33} height={33} pubkey={id} />
+        <div className={classes.address}>{account}</div>
+      </Link>
```
Comment (sunnygleason) [inline src/v2/components/Transactions/Detail/Application/index.jsx]: @manuel-calavera this looks good, just one background modeling question...

@CriesofCarrots we're treating apps and accounts the same here within transactions, does that matter?

is there a way to disambiguate, or is it program/instruction-specific whether the semantic is account or application?
(let us know if we should treat different program ids differently based on some heuristic)

Comment (CriesofCarrots) [reply src/v2/components/Transactions/Detail/Application/index.jsx]: From the chain perspective, a program is an account that has `"executable"=true`. So I think that we can disambiguate that way when loading an actual Application or Account Detail page.

However, currently (elsewhere) in the network explorer, I think that we're identifying applications based on appearance in the `program_id` field in a transaction, correct? (The alternative being tracking LoaderInstruction::Finalize instructions, which set that `executable` byte: https://github.com/solana-labs/solana/blob/bf9c815b9e9d8cf7909a530a611921ebda283fb7/sdk/src/loader_instruction.rs#L22)
If so, I think that for now, we ought to link ids in application fields to `/application/${id}` by default, and we can think about how to handle cases where the transaction-provided account isn't actually a program on the Application Detail page, when we load actual account data. That would mean that on Overview pages, an id's identification as program or application would be instruction specific. Thoughts?

File: src/v2/components/Account/Detail/index.jsx (hunk 5 above)
Comment (manuel-calavera) [reply src/v2/components/Account/Detail/index.jsx]: marked as to do for now

**Pull request merged**